terraform/modules/ecs/README.md

```diff
@@ -0,0 +1,304 @@
+<!-- BEGIN_TF_DOCS -->
+# Polytomic On ECS Fargate
+
+This module will all the necessary configuration to run a Polytomic On ECS Fargate.
+Using the architecture outlined in the image below.
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+
+# Architecture
+![arch](https://d2908q01vomqb2.cloudfront.net/1b6453892473a467d07372d45eb05abc2031647a/2018/01/26/Slide5-1024x647.png)
```

Comment:
(Just curious, is this a CF distribution we control? Or should the image live in the repo here?)

Reply:
(Maybe it's auto-generated, in which case who cares?)

Reply:
I stole it from this blog post: https://aws.amazon.com/de/blogs/compute/task-networking-in-aws-fargate/. I'll copy it in for now. We should be able to autogen the docs in the near future